1Password/shell-plugins
# new(redis-cli): Add support for Redis CLI with password as environment variable and command-line flags

## Overview


This PR adds support for redis-cli. The password for connecting to the redis server can be set as an environment variable and it'll be used by the shell plugin.

## Type of change
 


## Related Issue(s)
  

* Resolves: https://github.com/1Password/shell-plugins/issues/275

## How To Test


- Sign up for a Redis hosting provider like [Upstash](https://upstash.com/).
- Create a Redis server.
- Make note of the password, hostname and port number shown by that provider.
- Clone this branch and run `make redis/build`
- Set up redis shell plugin by running `op plugin init redis-cli`. Set the password as a 1Password field item.
- Run `redis-cli -h host-address-here.com -p port-number-here PING`
- See response `PONG`

<img width="1593" alt="image" src="https://github.com/1Password/shell-plugins/assets/18581859/f764380b-caae-4f6d-917f-20c0f41f421a">

## Changelog
  

Add support for Redis CLI with environment variable-based provisioning.
